.golangci.yml

```diff
@@ -5,13 +5,24 @@ run:
     - pkg/util/cloudproviders/cloudfoundry/bbscache_test.go # implements interface from imported package whose method names fail linting
 
 issues:
+  exclude-use-default: false
   # Do not limit the number of issues per linter.
   max-issues-per-linter: 0
 
   # Do not limit the number of times a same issue is reported.
   max-same-issues: 0
 
   exclude:
+    - "`RandomString|TimeNowNano|InitLogging|SetHostname` is unused" # 'deadcode' errors in test/benchmarks/util.go
+    - "Error return value of `io.WriteString` is not checked" # 'errcheck' errors in tools/dep_tree_resolver/go_deps.go
+    - "Error return value of `pem.Encode` is not checked" # 'errcheck' errors in test/integration/utils/certificates.go
+    - "exported: exported type EasyjsonTime|NewEasyjsonTime|EasyjsonTime.MarshalEasyJSON|EasyjsonTime.UnmarshalJSON should have comment or be unexported" # 'revive' errors in pkg/security/utils/json.go
+    - "exported: exported const Exec should have comment \\(or a comment on this block\\) or be unexported" # 'revive' errors in pkg/process/events/model/model_common.go
+    - "exported: exported const APIName should have comment \\(or a comment on this block\\) or be unexported" # 'revive' errors in pkg/serverless/trace/inferredspan/constants.go
+    - "unnecessary conversion" # 'unconvert' errors in test/integration/utils/certificates_test.go
+    - "SA1019*" # 'staticcheck' errors about SA1019
+    - "SA4011*" # 'staticcheck' errors about SA4011
```

Comment:
I think we should add this to the list for staticcheck checks at the very end of the file instead of here